metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -167,6 +168,8 @@ class MetalsLanguageServer(
   private val recentlyOpenedFiles = new ActiveFiles(time)
   private val recentlyFocusedFiles = new ActiveFiles(time)
   private val languageClient = new DelegatingLanguageClient(NoopLanguageClient)
+
+  @volatile
```

Comment:
Adding `@volatile` to `var userConfig` looks good 👍 

Just out of curiosity, did you find any troubles if we don't have `@volatile`? Or you added just in case?

Reply:
Yep - just in case)